tests/ui_tests/test_configs.py

```diff
@@ -359,39 +383,66 @@ def test_configs_fields(sdk_client_fs: ADCMClient, config_dict, app_fs):
+
     with allure.step('Check field configuration'):
         if expected['visible']:
             if expected['visible_advanced']:
-                assert not fields, "Config fields presented, expected no"
-                ui_config.show_advanced()
-                assert ui_config.advanced, 'Advanced button not clicked'
-            fields = ui_config.get_app_fields()
+                config_page.config.check_no_rows_or_groups_on_page()
+                config_page.config.click_on_advanced()
+            fields = config_page.config.get_all_config_rows()
             assert fields, 'No config fields, expected yes'
             for field in fields:
-                ui_config.assert_field_is_editable(field, expected['editable'])
+                expected_editable_state = expected['editable']
+                assert (
+                    config_page.config.is_element_editable(field) == expected_editable_state
+                ), f"Element should{' not ' if expected_editable_state is False else ' '}be editable"
             if expected['content']:
-                ui_config.assert_field_content_equal(cluster, field_type, fields[0], config['config'][0]['default'])
+                if field_type == 'boolean':
+                    config_page.config.assert_bool_value_is(field, expected_value=config['config'][0]['default'])
+                elif field_type in ("password", "secrettext"):
+                    is_password_value = True if field_type == "password" else False
+                    config_page.config.assert_input_value_is(
+                        expected_value='********', display_name=field_type, is_password=is_password_value, is_list=False
+                    )
+                elif field_type == "list":
+                    config_page.config.assert_input_value_is(
+                        expected_value=config['config'][0]['default'],
+                        display_name=field_type,
+                        is_password=False,
+                        is_list=True,
+                    )
+                elif field_type == "map":
+                    config_page.config.assert_input_value_is(
+                        expected_value=config['config'][0]['default'],
+                        display_name=field_type,
+                        is_password=False,
+                        is_list=False,
+                        is_map=True,
+                    )
+                elif field_type == "file":
+                    config_page.config.assert_input_value_is(expected_value="test", display_name=field_type)
+                else:
+                    expected_value = (
+                        str(config['config'][0]['default'])
+                        if field_type in ("integer", "float", "json")
+                        else config['config'][0]['default']
+                    )
+                    config_page.config.assert_input_value_is(expected_value=expected_value, display_name=field_type)
```

Comment:
I've mentioned it above: what's the behavior when all flags are false? It isn't obvious

Reply:
If I understand correctly [this](https://github.com/arenadata/adcm/pull/1616/files/018ced1e093714abcdb503ed6bfca99c951f2d30#diff-72eb9f33b0e9c971f07e75c7bf7f54208a79d5bc30e7939711ee43e48b2a6ac1R165) is the method we're talking about. In if-elif-else tree I see check on list, map, but not password, so I guess `else` will take both password and "all false" situations

Reply:
ah, I've missed that "is_password" is used alternatively. it's confusing. I suggest using the special value for "special type" fields (sort of enum that is passed to a single param)

Reply:
no, in get_input_value() we need flag is_password to get special field for search